daemon/cmd/daemon_main.go

```diff
@@ -109,6 +109,14 @@ var (
 				genMarkdown(cmd, cmdRefDir)
 				os.Exit(0)
 			}
+
+			// Open socket for using gops to get stacktraces of the agent.
+			if err := gops.Listen(gops.Options{}); err != nil {
+				errorString := fmt.Sprintf("unable to start gops: %s", err)
+				fmt.Println(errorString)
```

Comment:
nit: what do you think about just return one error here  (e.g. return fmt.Errorf("unable to start gops: %s", err)), and let Execute() method to print error and exit -1

Execute() method
```go
        ...
	if err := RootCmd.Execute(); err != nil {
		fmt.Println(err)
		os.Exit(-1)
	}
```

Reply:
oh ignore my above comment. Just notice that this is for Run but not RunE.